src/main/scala/com/amazon/deequ/analyzers/catalyst/StatefulKLLSketch.scala

```diff
@@ -0,0 +1,91 @@
+package org.apache.spark.sql
```

Comment:
Please format according to our [coding conventions](https://github.com/databricks/scala-style-guide)

Reply:
Added the license header.